action/messageHubFeed.js

```diff
@@ -120,6 +156,20 @@ function validateParameters(rawParams) {
         return;
     }
 
+    if(rawParams.kafka_admin_url) {
+        validatedParams.kafka_admin_url = rawParams.kafka_admin_url;
```

Comment:
@dubeejw The problem was that I was using both `json:true` in the request options _and_ calling `JSON.parse()` on the response body. In this way, `JSON.parse()` is trying to parse an already-parsed JSON object - not a string like it is expecting.

This should be fixed now.